debian/copyright

```diff
@@ -0,0 +1,43 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: font-downloader
+Upstream-Contact: Jonas Jelten <jj@sft.mx>
+Source: https://github.com/SFTtech/nyan
+
+Files: *
+Copyright:	2016-2022 Jonas Jelten <jj@sft.mx>
+		2016-2022 Markus Otto <otto@fs.tum.de>
+		2016-2022 Michael Enßlin <michael@ensslin.cc>
+		2016-2022 Andre Kupka <kupka@in.tum.de>
+		2016-2022 Tushar Maheshwari <tushar27192@gmail.com>
+		2016-2022 Andrew Thompson <mrwerdo331@me.com>
```

Comment:
i think as we have so many authors, a link to the author document would be better here

Reply:
Afaik you have to include all the authors even if there are a lot.

Reply:
hm, but then the list would be really long, and would outdate very quickly.
Can't we do `Copyright: 2013-2022 the openage authors` like we do in all source files?